File: Python Course/Software Development Module - in Python/Week11/Assessment.py
"""
Assessment.py
Billy Cussen
06/10/2020
"""

class myAssessment:
    
    def __init__(self, weighting, name, typeOfAssessment):
        self.weighting = weighting
        self.name = name
        self.typeOfAssessment = typeOfAssessment

    def setWeighting(self, weighting):
        self.weighting = weighting

    def setName(self, name):
        self.name = name

    def setType(self, typeOfAssessment):
        self.typeOfAssessment = typeOfAssessment

    def toString(self):
        return "Weighting: "+str(self.weighting) + ", Name: " + str(self.name)+", Type: " + str(self.typeOfAssessment)

File: Java/Software Development Module - Java/Week11/Assessment.java
/*
Assessment.java
Billy Cussen
21/07/2020
*/

public class Assessment {
  
    //Data Members
    public int weighting;
    public String name;
    public String type;

    //Constructors
    public Assessment(){
        this.weighting = 0;
        this.name = new String();
        this.type = new String();
    }

    public Assessment(int weighting, String name, String type){
        this.weighting = weighting;
        this.name = name;
        this.type = type;
    }

    //Setters
    public void setWeighting(int weighting){
        this.weighting = weighting;
    }

    public void setName(String name){
        this.name = name;
    }

    public void setType(String type){
        this.type = type;
    }
}